src/lib/Microsoft.Health.Fhir.Ingest.Common/Microsoft.Health.Fhir.Ingest.Common.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
You will want to set the default namespace to  Microsoft.Health.Fhir.Ingest.Data.

Reply:
Discussed offline, I'll make the change.